api/osrd_infra/schemas/infra.py

```diff
@@ -81,7 +81,7 @@ class DirectionalTrackRange(BaseModel):
 
     @root_validator
     def check_range(cls, v):
-        assert v.get("begin") < v.get("end"), "expected: begin < end"
+        assert v.get("begin") <= v.get("end"), "expected: begin < end"
```

Comment:
Do we really have ranges with begin == end? That should probably not be allowed.

But if it's valid then the error needs to be updated as well

Reply:
It's valid, this is why we have the `direction` field.
What do you mean by "error" ?

Reply:
The second half of the assert, `expected: begin < end`